### ISSUE TITLE
Add ability to describe sequence and numbering

### ISSUE THREAD

**Issue by kvanmalssen** [WGBH/PBCore2.0#78]
Intellectual content items are often season descriptions, episodes or parts of some other type of sequential arrangement. Currently, there is not simple way to express this in pbcore. It seems as though you could do this using titles, e.g.

```
<pbcoreTitle type='Series'>The Science Show</pbcoreTitle>
<pbcoreTitle type='Season'>Season 2</pbcoreTitle>
<pbcoreTitle type='Episode'>The Trouble with Herbivores</pbcoreTitle>
<pbcoreTitle type='Episode Number'>Episode 3</pbcoreTitle>

```

But this seems inelegant and prone to confusion and inconsistent use. In order to enable a more simply way to express this information, it might be worth considering a sequence or numbering element. EIDR achieves this by using a structure like:

```
  <EpisodeInfo>
      <Parent>10.5240/0AE0-E81D-9F5C-4C14-D13C-J</Parent>
      <SequenceInfo>
        <md:DistributionNumber domain="example.com">4</md:DistributionNumber>
        <md:AlternateNumber domain="example.com">104</md:AlternateNumber>
      </SequenceInfo>
    </EpisodeInfo>
```

**Comment by lelayz** [comment]
Good idea except that PBS does NOT number their segments within programs.  They don't even always give them separate titles.  And the segments share  a program number.

So the multiple use of titles is actually a good thing.

Otherwise we would be artificially assigning numbers to segments that would mean nothing except to the institution assigning it.

There are unique PBS package numbers that could be helpful?

Leah

**Comment by GlennClatworthy** [comment]
I haven’t caught up with recent threads but just noticed Leah’s comment.

Actually we do number segments for screening purposes at PBS. Most programs have only one, but we create separate segments for series like the NewsHour, Charlie Rose, and Tavis Smiley, for which the topics vary considerably from segment to segment. That’s why I added issue number 68.

**Comment by lelayz** [comment]
What about the magazine shows: Nova ScienceNow and Frontline?  I don't see segment numbers in the PBS descriptions or databases, but if they exist that would be a great help! Just let me know where to find them. Leah

**Comment by AllisonAnn** [comment]
Is this also related to issue #63?

Could an "otherNumber" element/container (separate from pbcoreIdentifier) be useful to house such additional numbers associated with the asset?  

We could recommend a @type vocab (DistributionNumber | AlternateNumber | SequenceNumber...)- or leave it to the local institution to decide the type value.  I would also recommend a note element for added information for the otherNumber.

Having an OtherNumber container would be useful for a lot of purposes - not just to house sequence numbers.  And, it would serve as a central location to house all additional numbers associated with an asset, and keep us from extending PBcore in an unwieldy way.

**Comment by GlennClatworthy** [comment]
Most of those get single segments.

We create multiple segments for the nightly and weekly news programs to support FCC public file reporting. Stations need to provide quarterly lists of programs they have aired covering issues of potential interest to their local communities.

If segment one is about the Obama Administration’s plan for the economy, for example, and segment two is about climate change, I would code segment one as “Economy” and "National Politics/Government" and segment two as “Environment.”

Segment numbers are provided to stations via our Program Offer Data Service feeds. I don’t know how they get translated in ProTrack.

Glenn

**Comment by lelayz** [comment]
Thanks, Glenn!  Now I need to get the productions to refer to them that way! Leah

**Comment by GlennClatworthy** [comment]
Here for reference are two examples from my own experiments in creating PBS program records in RDF:

http://dreampublic.net/pbs/rdf/1.2/programs/M/L/MLNH009511.xml features a general length and specific lengths for individual segments.

http://dreampublic.net/pbs/rdf/1.2/programs/A/B/ABML000103.xml features a generic length for the single segment.

[Note that these links may change. I'm sorting out inconsistent behavior between "dreampublic.net" and "www.dreampublic.net".]

The precise length of the entire program is stored in the records for individual packages (i.e., instantiations), which I haven't covered here, so in that respect I think the PBS Broadview model matches PBCore.
